docs/modules/ROOT/pages/how-to/teams.adoc

```diff
@@ -0,0 +1,35 @@
+= Managing Teams
+
+The {product} Portal allows users to create teams, and assign users to them.
+
+In order to be able to assign users to teams, they need to xref:how-to/find-username.adoc[know their username].
+
+== Creating a Team
+
+To create a team, click the menu:Teams[] item on the navigation, and then click on the btn:[Add new Team] button.
```

Comment:
Important. The team will be created for the organization selected (organization selector at the top).

Reply:
Fixed